### PR TITLE
Add support for Rails 6.1.0

### PR DESCRIPTION
**Pull request by drtechie**
Changes made

 - Changed AR dependency to < 6.2.0
 - Added necessary changes in travis.yml
   - Added Ruby version 2.7.3
   - Excluded AR <= 5.2 from 2.7.3 tests.

Addresses: #165

### CODE REVIEW

**Comment by smridge** [comment]
@thuss any updates on getting this merged with a new version release? I recently ran into this blocking an `activerecord` update.

File: standalone_migrations.gemspec [hunk 1]
```diff
@@ -60,12 +60,12 @@ Gem::Specification.new do |s|
...
-    s.add_runtime_dependency(%q<activerecord>.freeze, [">= 4.2.7", "< 6.1.0", "!= 5.2.3", "!= 5.2.3.rc1"])
-    s.add_runtime_dependency(%q<railties>.freeze, [">= 4.2.7", "< 6.1.0", "!= 5.2.3", "!= 5.2.3.rc1"])
+    s.add_runtime_dependency(%q<activerecord>.freeze, [">= 4.2.7", "< 6.2.0", "!= 5.2.3", "!= 5.2.3.rc1"])
+    s.add_runtime_dependency(%q<railties>.freeze, [">= 4.2.7", "< 6.2.0", "!= 5.2.3", "!= 5.2.3.rc1"])
   else
     s.add_dependency(%q<rake>.freeze, [">= 10.0"])
-    s.add_dependency(%q<activerecord>.freeze, [">= 4.2.7", "< 6.1.0", "!= 5.2.3", "!= 5.2.3.rc1"])
-    s.add_dependency(%q<railties>.freeze, [">= 4.2.7", "< 6.1.0", "!= 5.2.3", "!= 5.2.3.rc1"])
+    s.add_dependency(%q<activerecord>.freeze, [">= 4.2.7", "< 6.2.0", "!= 5.2.3", "!= 5.2.3.rc1"])
+    s.add_dependency(%q<railties>.freeze, [">= 4.2.7", "< 6.2.0", "!= 5.2.3", "!= 5.2.3.rc1"])
```
Comment (smridge) [inline standalone_migrations.gemspec:68]: Thoughts on relaxing this and the others to `< 7.0`?

Comment (thuss) [reply standalone_migrations.gemspec:68]: That would include future versions that we haven't tested yet so we try and keep it current to the patch level until we can test.

**Comment by thuss** [comment]
I must have missed this PR. Thanks for the PR @drtechie and the reminder @smridge

**Comment by thuss** [comment]
I've published version 6.1.0 of standalone_migrations to ruby gems. Thanks!

**Comment by smridge** [comment]
Thank you for the quick turnaround on this @thuss ! Kudos to @drtechie as well 🙌🏻

**Pull request merged**